docs/content/docs/build/audience.md

```diff
@@ -78,41 +79,40 @@ The `getMyself` method returns the `IMember` object from the audience correspond
 
-#### Presence with No Shared Data
+#### Unshared Data
 
-Not all presence scenarios will require sharing data among users - the data could be generated locally or fetched from an external service.  Consider the scenario where you want to display the connected users with a profile picture and a color border.  If you retrieve a user's profile picture from your service and assign each user a color based on a hash of their ID, you will have data on other connected users without needing to communicate with them.
+Not all presence scenarios will require sharing data among users - the data could be generated locally or fetched from an external service.  Consider the scenario where you want to display the connected users with a profile picture and a color border.  If you retrieve a user's profile picture from your service and assign each user a color based on a hash of their ID, you will have the desired data on other users without needing to communicate with them.
 
-#### Presence with Persisted Data
+#### Shared Persisted Data
 
-Most presence scenarios will involve data that only a single user or client knows and will need to communicate to other audience members.  Some of those scenarios will require you to save data for each user for future sessions.  Consider if you want to display a live counter of how long each user has spent in your application.  You want to be able to see an active user's counter ticking up, and when that user disconnects you want to be able to resume that counter once they reconnect.  You may choose to use a `SharedMap` with a `SharedCounter` as the value onto which each user will increment their time spent every minute (see [Introducing Distributed Data Structures](./dds.md)).  All other connected users will then receive changes to that map automatically.  Your UI can display data from the map for only users present in the audience.  A returning user can find themselves in the map and resume from the latest state.
+Most presence scenarios will involve data that only a single user or client knows and needs to communicate to other audience members.  Some of those scenarios will require you to save data for each user for future sessions.  Consider if you want to display how long each user has spent in your application.  An active user's time should tick up, pause when they disconnect, and resume once they reconnect.  You may choose to use a `SharedMap` with a `SharedCounter` as the value onto which each user will increment their time spent every minute (also see [Introducing Distributed Data Structures](./dds.md)).  All other connected users will then receive changes to that map automatically.  Your UI can display data from the map for only users present in the audience.  A returning user can find themselves in the map and resume from the latest state.
```

Comment:
```suggestion
Most presence scenarios will involve data that only a single user or client knows and needs to communicate to other audience members.  Some of those scenarios will require you to save data for each user for future sessions.  For example, consider a scenario where you want to display how long each user has spent in your application.  An active user's time should increment while connected, pause when they disconnect, and resume once they reconnect.  This means that the time each user has spent must be persisted so it can survive disconnections.

One option is to use a `SharedMap` with a `SharedCounter` as the value onto which each user will increment their time spent every minute (also see [Introducing distributed data structures]({{< relref dds.md >}})).  All other connected users will then receive changes to that SharedMap automatically.  Your UI can display data from the map for only users present in the audience.  A returning user can find themselves in the map and resume from the latest state.
```